network/wsPeer.go

```diff
@@ -343,6 +347,8 @@ func (wp *wsPeer) writeLoopSend(msg sendMessage) (exit bool) {
 		// just drop it, don't break the connection
 		return false
 	}
+	atomic.StoreInt64(&wp.intermittentOutgoingMessageEnqueueTime, msg.enqueued.UnixNano())
```

Comment:
maybe we should check at this moment if the message we're about to send is too old, like in wsPeer.CheckSlowWritingPeer(). This closes a window where between packets that CheckSlowWritingPeer() would close, the peer will survive.

Reply:
Makes sense. I'll add another check here as well.